webapp/lib/controller.dart

```diff
@@ -1002,27 +1159,61 @@ void sendReply(model.SuggestedReply reply, model.Conversation conversation) {
       conversation.docId,
       conversation.messages.indexOf(newMessage),
       translation: newMessage.translation,
+      status: newMessage.status,
       incoming: false);
   view.conversationPanelView.addMessage(newMessageView);
+
+  if (!queuedMessagesPerConversation.containsKey(conversation)) {
+    queuedMessagesPerConversation[conversation] = [];
+  }
+  queuedMessagesPerConversation[conversation].add(newMessage);
```

Comment:
If you prefer, these lines can be changed to
```
queuedMessagesPerConversation.putIfAbsent(conversation, () => []).add(newMessage)
```

Reply:
nice, thanks for the suggestion, done!